### DISCUSSION TITLE
Globe Icon

### DISCUSSION THREAD

**Discussion by hughrawlinson** [twilio-labs/paste#2580] [Request: Icon] [answered]
Hi, there's an icon in the twilio.com (and twilio.com/docs, and a few other places) in the top nav, in the language picker - a globe. That icon isn't available in Paste (as far as I can see) - could we get that icon (or is there a better icon I should be using for that)

**Comment by serifluous** [comment]
Hey @hughrawlinson ! Could you share what use case or a screenshot of what you're looking to use the icon for?

We currently have a globe icon called the ProductRegionalIcon. You can find it here: https://paste.twilio.design/components/icons/

It's meant to be used specifically to indicate the Regional/Global product. If you're using it for other purposes, we could consider adding a generic UI icon for it. Let us know!

**Reply by hughrawlinson** [reply]
Hey, thanks for getting back to me. I'm using it for exactly the same purpose as on the original site - I'm building a copy of the twilio.com/docs site in React, and I am trying to make as few UI changes as possible while adopting as much of Paste as possible. We don't unfortunately have a designer working on this project, so I don't really have anyone to ask if it's ok to use this icon instead. I can use it in development, but I think since we don't have anyone to approve a change, it would be best to keep the experience consistent with the rest of twilio.com. What do you think is the best approach here?

**Reply by serifluous** [reply]
Got it—we can add this icon to Paste in our next release. There's a chance we might alter it slightly so it clearly communicates a language switch and will be different enough from the existing ProductRegionalIcon, but we'll reach out to you with the design.

The work is ticketed here if you want to follow along: https://issues.corp.twilio.com/browse/DSYS-3332.

**Reply by serifluous** [reply]
Hey @hughrawlinson , as a heads up, we'll be adding this `TranslationIcon` to Paste soon for this use case. After chatting with the team, we didn't want it to be too easily confused with Global/Regional products or features. Hope this slight difference is ok between twilio.com and twilio.com/docs. I think the language selector next to the icons is potentially obvious enough for customers to understand what it is, despite the icon difference, but definitely let us know if you have concerns. Thanks!

![Screen Shot 2022-08-30 at 2 29 39 PM](https://user-images.githubusercontent.com/3821840/187546893-2f160828-79b9-4460-bf0f-d9340f732173.png)

**Reply by serifluous** [reply]
Rough mockup in context:
![image](https://user-images.githubusercontent.com/3821840/187548851-02e58e6d-72a3-4205-999c-20455cd3746c.png)

**Reply by hughrawlinson** [reply]
That looks great to me! Thanks :)

**Comment by SiTaggart** [comment, marked as answer]
Hi @hughrawlinson 

The Translate icon just landed in NPM! https://paste.twilio.design/components/icons

**Reply by hughrawlinson** [reply]
Sweet! Thanks 😄
